client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -24,12 +24,7 @@ import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import CountedTextarea from 'components/forms/counted-textarea';
 import Banner from 'components/banner';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
-import {
-	getSeoTitleFormatsForSite,
-	isJetpackMinimumVersion,
-	isJetpackSite,
-	isRequestingSite,
-} from 'state/sites/selectors';
+import { getSeoTitleFormatsForSite, isJetpackSite, isRequestingSite } from 'state/sites/selectors';
```

Comment:
`isRequestingSite`/`isFetchingSite` should always return `false` and are redundant.

Reply:
Good find, removed in https://github.com/Automattic/wp-calypso/pull/30039/commits/7086328a0b674de741a7e6c33c0616b405609c08.

Reply:
Turns our I was wrong about `isFetchingSite`. On initial mount and after successfully saving the edited form, the `refreshCustomTitles` method will fetch the latest version of the `site` object from server. While that request is in flight, `isFetchingSite` will be `true` and the component state transitions are affected by that.

I don't understand why the `invalidatedSiteObject` field is being set on the state. It's never read, only written. ❓

Reply:
Oh, right, I remember now 👍 Okay, I've reverted the `isRequestingSite` logic in https://github.com/Automattic/wp-calypso/pull/30039/commits/9beead9320be93cfb6ac782559de1d43efc7c484

And on `invalidatedSiteObject` - I'm not sure either, but I prefer leaving it as-is. The whole flow of this (and several other SEO-related components) need some more special attention to clean any other oddities and code smells that currently exist.